### PR TITLE
Fix issue 11708

### PR DESCRIPTION
**Pull request by Rolanddoda**
Link to the issue: https://github.com/quasarframework/quasar/issues/11708

**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Documentation
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] It's submitted to the `dev` branch (or `v[X]` branch)
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested on a Cordova (iOS, Android) app
- [ ] It's been tested on a Electron app
- [ ] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar/tree/dev/docs) (for faster update click on "Suggest an edit on GitHub" at bottom of page) or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: app/lib/webpack/create-chain.js [hunk 1]
```diff
@@ -211,7 +211,7 @@ module.exports = function (cfg, configName) {
       .options({
         esModule: false,
         limit: 10000,
-        name: `img/[name]${fileHash}.[ext]`
+        name: `img/[path][name]${fileHash}.[ext]`
```
Comment (rstoenescu) [inline app/lib/webpack/create-chain.js:214]: Hi,

Thanks for contributing!
Can you add `[path]` for the fonts too pls? (It's right below the images section)

**Review by rstoenescu — CHANGES_REQUESTED** [review]

**Comment by rstoenescu** [comment]
Finally got a chance to test it out.
Thanks for contributing! This will go into next q/app.

**Comment by Alexssmusica** [comment]
Good afternoon, this change caused a problem when in production, it doesn't find the files. In development it is normal.

**Comment by rstoenescu** [comment]
@Alexssmusica can you provide a minimal reproduction repo pls? From our testing nothing seems to be wrong.

**Comment by Alexssmusica** [comment]
@rstoenescu 
It actually happens when you use these settings in quasar.conf
image

**Comment by Alexssmusica** [comment]
@rstoenescu 
I use electron, after compiling it doesn't recognize the icons.
image

**Comment by Alexssmusica** [comment]
@rstoenescu 
And when spa this happens.
image

**Comment by Alexssmusica** [comment]
@rstoenescu 
In my project I went back to version 3.2.9, it worked again. These prints were a test done in the standard quasar create project.

**Comment by Alexssmusica** [comment]
@rstoenescu https://github.com/Alexssmusica/test-quasar-app-3.3.0.git

**Comment by rstoenescu** [comment]
Hmm, seems like it actually breaks even more use-cases. Reverted and published q/app v3.3.1

**Comment by Alexssmusica** [comment]
@rstoenescu  Thanks!!!

**Comment by rstoenescu** [comment]
Now need to find a way to solve the initial problem though

**Comment by Rolanddoda** [comment]
@rstoenescu should you open issue #11708 after merging this?

**Comment by rstoenescu** [comment]
> @rstoenescu should you open issue #11708 after merging this?

Can't reopen since it was changed into a discussion.

**Comment by Rolanddoda** [comment]
@rstoenescu I see. Should I open a new issue though?

**Comment by rstoenescu** [comment]
No need. I'm actively working on a solution with minimal perf impact. The last part is the hardest one.

**Comment by Rolanddoda** [comment]
@rstoenescu ahh cool, good to know. Thank you and good luck with the last part

**Comment by rstoenescu** [comment]
Will be available in q/app v3.3.2

**Pull request merged**